packages/react/src/components/dropzone.tsx

```
@@ -126,590 +91,166 @@ export function UploadDropzone<
+                  <span className="z-50">
+                    <span className="block group-hover:hidden">
+                      {uploadProgress}%
+                    </span>
+                    <Cancel
+                      cn={cn}
+                      className="hidden size-4 group-hover:block"
+                    />
+                  </span>
+                );
+              }
+              case "disabled":
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)